### PR TITLE
Commander arming checks

### PR DESCRIPTION
**Pull request by LorenzMeier**
I stumbled into these while working on a bench setup and this helps with more consistent behavior.

### CODE REVIEW

File: src/modules/commander/Arming/PreFlightCheck/checks/ekf2Check.cpp [hunk 1]
```diff
@@ -242,7 +242,7 @@ bool PreFlightCheck::ekf2CheckSensorBias(orb_advert_t *mavlink_log_pub, const bo
 
 		// check accelerometer bias estimates
 		if (bias.accel_bias_valid) {
-			const float ekf_ab_test_limit = 0.5f * bias.accel_bias_limit;
+			const float ekf_ab_test_limit = 0.75f * bias.accel_bias_limit;
```
Comment (dagar) [inline src/modules/commander/Arming/PreFlightCheck/checks/ekf2Check.cpp:245]: I'd be fine with this change, but what do you think @bresch?

File: src/modules/commander/commander_params.c [hunk 2]
```diff
@@ -307,6 +307,8 @@ PARAM_DEFINE_FLOAT(COM_DISARM_PRFLT, 10.0f);
  * The default allows to arm the vehicle without GPS signal.
  *
  * @group Commander
+ * @value 0 Allow arming without GPS
+ * @value 1 Require GPS lock to arm
  * @boolean
```
Comment (dagar) [inline src/modules/commander/commander_params.c:312]: Remove boolean.

File: src/modules/commander/Arming/PreFlightCheck/checks/ekf2Check.cpp (hunk 1 above)
Comment (dagar) [reply src/modules/commander/Arming/PreFlightCheck/checks/ekf2Check.cpp:245]: With default values this is an accel bias of 0.3 m/s/s.

Comment (bresch) [reply src/modules/commander/Arming/PreFlightCheck/checks/ekf2Check.cpp:245]: I'm fine with increasing that check unless we start to see issues due to large incorrect accel bias estimate on takeoff.

**Review by bresch — DISMISSED** [review]

**Comment by dagar** [comment]
Github actions is having a bad day, restarting again...

**Pull request merged**